docs/scarpet/api/Entities.md

```diff
@@ -400,6 +400,30 @@ query(p,'effect','resistance')  => null
 
 Number indicating remaining entity health, or `null` if not applicable.
 
+### `query(e, 'may_fly')`
+
+Boolean. Returns the value of player's `mayFly` ability.
```

Comment:
For consistency, around the docs there's usually not a type -> what, but a slightly more descriptive without saying "this is a float" or "this is an int" description. Mostly since in scarpet there's not really a differentiation between integer, float, double or long (at a language level, internally of course there's some).

For booleans to keep a bit of consistency with the rest of the docs you could keep it like "Returns a boolean indicating the value of the player's `flying` ability" for example but for numbers (floats in this case) don't specify the type, so maybe "Returns a number being the value of the player's `flySpeed` ability", though you don't need to specify it's a number if it's obvious, and (while in this case I don't really know how it could be replaced cleanly, so I'd keep it as is) it's better to not specify implementation details like "where does this exactly come from", mostly cause the api is expected to work even if the exact names/impl details change.